xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/lightbox/lightbox.css

```diff
@@ -67,7 +67,7 @@
 #lb {
   left: 0;
   margin-left: 0;
-  width: auto;
+  width: 100%;
```

Comment:
Why is this needed? The value was reset in order to fix XWIKI-10734. 
Would be nice to see how it affects the responsiveness of Modals.

Also I think width:auto rule should have been removed, since that's the default.